src/App.js

```diff
@@ -27,7 +28,8 @@ function App(){
             </div>
             <div className="m-auto pt-20 w-full">
                 <Routes>
-                  <Route path="/" element={ <LeaderBoard/> } />
+                  <Route path="/" element={<Loading/>} />
```

Comment:
This was done so that the information can be loaded from the subgraph first and then displayed on the leaderboard.